packages/core/README.md

```diff
@@ -166,3 +166,32 @@ yarn optimsim-up # start the optimism containers. note this will take a long tim
 cd ./packages/core # move back to this package
 yarn test-e2e # run the end to end tests against the optimism containers.
 ```
+
+## Deploying the contracts
+
+To deploy the contracts, we use the `hardhat-deploy` package. This may be failiar to users of `hardhat`. The deployment
+process is faily simple, as a result.
+
+To deploy the entire UMA system on a network of your choice:
+
+```sh
+MNEMONIC="Your 12-word phrase here" CUSTOM_NODE_URL="Your node url here" yarn hardhat deploy --network kovan
```

Comment:
it might be easier to read this if you split these commands onto multiple lines.

Reply:
Done!